From 3c3046e3405ea053ab2e8f8fc14e4ed3cac6af38 Mon Sep 17 00:00:00 2001
From: Tri Luong <tri.luong.te@rvc.renesas.com>
Date: Tue, 26 Feb 2019 13:13:28 +0700
Subject: [PATCH 053/175] arm64: dts: renesas: r8a774c0: Add watchdog support

Add watchdog support to the RZ/G2E (a.k.a. R8A774C0) SoC
specific device tree.

Signed-off-by: Tri Luong <tri.luong.te@rvc.renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774c0.dtsi | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a774c0.dtsi b/arch/arm64/boot/dts/renesas/r8a774c0.dtsi
index 3c53fdd..30b524d 100644
--- a/arch/arm64/boot/dts/renesas/r8a774c0.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a774c0.dtsi
@@ -561,5 +561,15 @@
 			i2c-scl-internal-delay-ns = <6>;
 			status = "disabled";
 		};
+
+		rwdt: watchdog@e6020000 {
+			compatible = "renesas,r8a774c0-wdt",
+				     "renesas,rcar-gen3-wdt";
+			reg = <0 0xe6020000 0 0x0c>;
+			clocks = <&cpg CPG_MOD 402>;
+			power-domains = <&sysc R8A774C0_PD_ALWAYS_ON>;
+			resets = <&cpg 402>;
+			status = "disabled";
+		};
 	};
 };
-- 
2.7.4

